package.json

```diff
@@ -7,7 +7,8 @@
     "build": "babel src/ -d lib/ && flow-copy-source src lib",
     "prepublish": "npm run build",
```

Comment:
tomei um warning aqui
`npm WARN prepublish-on-install As of npm@5, `prepublish` scripts are deprecated.`

achei mais sobre isso aqui
https://docs.npmjs.com/misc/scripts

Reply:
Isso não é do meu PR

Reply:
muda de prepublish pra prepack que funciona (quase ctz)

Reply:
vou abrir uma nova issue

Reply:
quando que tu tomou o warning? 'npm install'?

Reply:
#15